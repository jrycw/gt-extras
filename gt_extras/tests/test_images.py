# from great_tables import GT
# import pytest
# from gt_extras.images import img_header


# def test_img_header_basic(mini_gt):
#     pass
