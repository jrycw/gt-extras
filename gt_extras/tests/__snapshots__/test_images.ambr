# serializer version: 1
# name: test_img_header_snapshot
  Html(text='<div style="text-align:center;">\n        <img src="https://example.com/image.png" style="\n        height:60px;\n        object-fit:contain;\n        object-position: bottom;\n        border-bottom:2px solid black;"\n    />\n        <div style="\n        font-size:12px;\n        color:black;\n        text-align:center;\n        width:100%;\n    ">\n        Test Label\n    </div>\n    </div>')
# ---
